### PR TITLE
about section fix in footer

### PR DESCRIPTION
**Pull request by udaymewada**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?

### New Feature Submissions:

1. [x] Does your submission pass build tests locally?
2. [x] Have you indented your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by monsij** [comment]
Hope that you have followed the contribution guidelines. Please don't make a request anywhere to get a review. One of the maintainers will get back to it within a day or so. 😃

(Powered by [GitMate.io](https://gitmate.io))

**Comment by udaymewada** [comment]
screenshot 65

**Comment by udaymewada** [comment]
#67

**Comment by monsij** [comment]
@udaymewada Please resolve conflicts to get the PR merged

**Comment by udaymewada** [comment]
done @monsij

**Comment by monsij** [comment]
@udaymewada : Give updated screenshots

**Comment by udaymewada** [comment]
screenshot 68

File: pro1/blog/templates/blog/base.html [hunk 1]
```diff
@@ -92,76 +92,73 @@
...
+								Register yourself to get started.</p>
+			  
+					  </div>
+					  <div class="col-md-4">
+			  
+						  <!-- Links -->
+						  <h5 class="text-uppercase">Important Links</h5>
+							<hr>
+						  <ul class="list-unstyled">
+								<li><a href="https://raw.githubusercontent.com/NIT-DGPortal/portal-files/master/Academic-Calendar.pdf" target ="_blank">
```
Comment (monsij) [inline pro1/blog/templates/blog/base.html]: Correct the link: It must be as per the previous one

File: pro1/blog/templates/blog/base.html [hunk 2]
```diff
@@ -92,76 +92,73 @@
...
+									Academic Calendar
+								</a></li>
+								
+								<li><a href="https://raw.githubusercontent.com/NIT-DGPortal/portal-files/master/Institute-Holidays.pdf" target = "_blank">
```
Comment (monsij) [inline pro1/blog/templates/blog/base.html]: Correct the link: It must be as per the previous one

File: pro1/blog/templates/blog/base.html [hunk 3]
```diff
@@ -92,76 +92,73 @@
...
+									Holidays
+								</a></li>
+								
+								<li><a href="http://172.16.24.9:9001/default.aspx" target ="_blank">
```
Comment (monsij) [inline pro1/blog/templates/blog/base.html]: Insert the updated Chanakya link

**Comment by udaymewada** [comment]
Ok

File: pro1/blog/templates/blog/base.html (hunk 3 above)
Comment (udaymewada) [reply pro1/blog/templates/blog/base.html]: done

**Comment by monsij** [comment]
Will merge soon!

**Comment by monsij** [comment]
ack `14b46d9`

**Comment by monsij** [comment]
@harry-hov  : Check and approve

**Comment by udaymewada** [comment]
Changes done @monsij @harry-hov

**Comment by monsij** [comment]
ack `b9e9ae3`

**Pull request merged**